images/base/entrypoint

```diff
@@ -19,12 +19,15 @@ set -o nounset
 set -o pipefail
 
 configure_proxy() {
+  # Don't use proxy for the local network
+  LOCAL_NETWORK=$(ip route list dev eth0 proto kernel | awk '{ print $1 }')
+
   mkdir -p /etc/systemd/system/containerd.service.d/
   cat <<EOF >/etc/systemd/system/containerd.service.d/http-proxy.conf
 [Service]
 Environment="HTTP_PROXY=${HTTP_PROXY:-}"
 Environment="HTTPS_PROXY=${HTTPS_PROXY:-}"
-Environment="NO_PROXY=${NO_PROXY:-}"
+Environment="NO_PROXY=${NO_PROXY:-localhost},${LOCAL_NETWORK}"
```

Comment:
- probably this should be localhost and 127... ?
- do we expect the user to ever want to be able to not NO_PROXY LOCAL_NETWORK for some reason?
- do we ever expect to want to proxy localhost / 127?

Reply:
I wouldn´t be surprised if there is some application in the node trying to reach localhost and the connection get proxied, maybe etcd or kubelet, I think that´s harmless anyway and it simplifies the script using them as default value.

LOCAL_NETWORK is directly connected, also is a docker network only with containers. I can´t see any reason to use a proxy to reach those IPs (however, I´m sure some user will find an exception ;) )

Reply:
OK, SGTM :-)